tracer/src/Datadog.Trace.Bundle/Datadog.Trace.Bundle.targets

```diff
@@ -0,0 +1,77 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="RemoveDatadogNuGetFiles" AfterTargets="Publish" Condition="'$(RemoveUnusedDatadogFiles)' == 'true'">
+    <PropertyGroup>
+      <!-- Root where Datadog bundle files are published -->
+      <BundleRoot>$(PublishDir)datadog\</BundleRoot>
+
+      <!-- Helper properties -->
+      <IsNetFramework>$(TargetFramework.StartsWith('net4'))</IsNetFramework>
+
+      <!-- Map TargetFramework to required folder -->
+      <EffectiveTargetFramework Condition="$(IsNetFramework)">net461</EffectiveTargetFramework>
+      <EffectiveTargetFramework Condition="'$(TargetFramework)' == 'netstandard2.0'">netstandard2.0</EffectiveTargetFramework>
+      <EffectiveTargetFramework Condition="'$(TargetFramework)' == 'netcoreapp2.0'">netstandard2.0</EffectiveTargetFramework>
+      <EffectiveTargetFramework Condition="'$(TargetFramework)' == 'netcoreapp2.1'">netstandard2.0</EffectiveTargetFramework>
+      <EffectiveTargetFramework Condition="'$(TargetFramework)' == 'netcoreapp3.0'">netstandard2.0</EffectiveTargetFramework>
+      <EffectiveTargetFramework Condition="'$(TargetFramework)' == 'netcoreapp3.1'">netcoreapp3.1</EffectiveTargetFramework>
+      <EffectiveTargetFramework Condition="'$(TargetFramework)' == 'net5.0'">netcoreapp3.1</EffectiveTargetFramework>
+      <EffectiveTargetFramework Condition="'$(TargetFramework)' == 'net6.0'">net6.0</EffectiveTargetFramework>
+      <EffectiveTargetFramework Condition="'$(TargetFramework)' == 'net7.0'">net6.0</EffectiveTargetFramework>
+      <EffectiveTargetFramework Condition="'$(TargetFramework)' == 'net8.0'">net6.0</EffectiveTargetFramework>
+      <EffectiveTargetFramework Condition="'$(TargetFramework)' == 'net9.0'">net6.0</EffectiveTargetFramework>
```

Comment:
How do you think we can future proof this? For example, if we were to merge a PR today to add a new build for net9.0, this bundle logic would still include the net6.0 assets for applications targeting net9.0+ until we update this logic

Reply:
Hmm. .NET 5 and above a consistent naming pattern, so we could look at the existing directories and figure out the best match on-the-fly.

```
if $(TargetFramework) matches "net{major}.0" then
    keep directory named "net{major}.0" with the largest {major} <= $(TargetFramework)
```

Reply:
> if we were to merge a PR today to add a new build for net9.0, this bundle logic would still include the net6.0 assets for applications targeting net9.0+ until we update this logic

I'm not entirely averse to "updating this logic" being one of the requirements for adding a new target. Adding a new target is a _ton_ of work already 🤷‍♂️ 

However, we _do_ target new TFMs more regularly, and so I'm more interested in not having to update this with `net10.0` `net11.0` etc 🤔